Tasks/JavaToolInstallerV0/javatoolinstaller.ts

```diff
@@ -15,9 +16,11 @@ taskLib.setResourcePath(path.join(__dirname, 'task.json'));
 
 async function run(): Promise<void> {
     try {
-        let versionSpec = taskLib.getInput('versionSpec', true);
+        const versionSpec = taskLib.getInput('versionSpec', true);
+        const jdkArchitectureOption = taskLib.getInput('jdkArchitectureOption', true);
```

Comment:
Could you please pass this variable to `getJava()` to avoid double `getInput()` calling for `jdkArchitectureOption` input?

Reply:
Done.